core/src/Network/GRPC/Unsafe.chs

```diff
@@ -189,14 +189,6 @@ castPeek p = do
    useAsCString* `ByteString', useAsCString* `ByteString', `CTimeSpecPtr',unReserved `Reserved'}
   -> `Call'#}
 
--- | Create a channel (on the client) to the server. The first argument is
--- host and port, e.g. @"localhost:50051"@. The gRPC docs say that most clients
--- are expected to pass a 'nullPtr' for the 'ChannelArgsPtr'. We currently don't
--- expose any functions for creating channel args, since they are entirely
--- undocumented.
-{#fun grpc_insecure_channel_create as ^
```

Comment:
Do we need to ifdef these for back compatibility?

Reply:
I don't think so. Those are internal and not part of the public API.

Reply:
That make sense to me. We also let the user link in their own version of grpc, does that matter?

Reply:
Good point, there is a warning in README about that. It should be updated.

Reply:
Ok added a note. Thank you!